great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -383,7 +442,10 @@ def reconcile_profiler_rules(
 
     @staticmethod
     def _reconcile_rule_config(
-        existing_rules: Dict[str, Rule], rule_name: str, rule_config: dict
+        existing_rules: Dict[str, Rule],
+        rule_name: str,
+        rule_config: dict,
+        reconciliation_directives=DEFAULT_RECONCILATION_DIRECTIVES,
```

Comment:
Could we annotate the new arg please?

Reply:
@cdkini -- absolutely! DONE (in next commit). Thanks!